### ISSUE TITLE
Neural networks potential example and optax import

### ISSUE THREAD

**Issue by cwbeitel** [google/jax-md#121]
Looks like the [neural network potentials notebook](https://github.com/google/jax-md/blob/master/notebooks/neural_networks.ipynb) needs to be updated for the new location of [optax](https://github.com/deepmind/optax) given error produced in "Imports & Utils":

```
     21 
     22 from jax import random
---> 23 from jax.experimental import optix
     24 
     25 from jax_md import energy, space, simulate, quantity

ImportError: cannot import name 'optix'
```

Fixed by adding `!pip install -q git+https://www.github.com/deepmind/optax` and replacing `from jax.experimental import optix` with `import optax as optix` without modifying the rest, alternatively could go through and update all the other usages of "optix".

Lmk if you want a patch or would rather do it. Awesome notebook btw! 😀

**Comment by cwbeitel** [comment]
Oh also later in the notebook the second to last cell gives an error
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-25-43e0542c4175> in <module>()
     17 # Initialize the simulation
     18 
---> 19 state = init_fn(key, test_positions[0], Si_mass)
     20 nbrs = neighbor_fn(state.position)
     21 

12 frames
/usr/local/lib/python3.6/dist-packages/haiku/_src/transform.py in apply_fn(params, state, rng, *args, **kwargs)
    295         rng, err_msg=(APPLY_RNG_STATE_ERROR if state else APPLY_RNG_ERROR))
    296     with base.new_context(params=params, state=state, rng=rng) as ctx:
--> 297       out = f(*args, **kwargs)
    298     return out, ctx.collect_state()
    299 

TypeError: model() missing 1 required positional argument: 'neighbor'
```

which is fixed by adding a neighbor parameter `state = init_fn(key, test_positions[0], Si_mass, neighbor=neighbor)`.

**Comment by sschoenholz** [comment]
Thanks a lot for the report! Should be fixed as of e6cc345.
